fastats/maths/__init__.py

```diff
@@ -31,4 +32,6 @@
     't_statistic',
     'relu',
     'softplus',
+    'ols_cholesky',
+    'ols_svd'
```

Comment:
Have you done any benchmarking on these? With and without JIT? It's super useful information, would be nice to have a notebook with benchmarks.

Reply:
They don't JIT, but they have quite different performance characteristics - it's a good point